editor.planx.uk/src/@planx/components/shared/Buttons/ImageRadio.tsx

```diff
@@ -0,0 +1,144 @@
+import ImageIcon from "@mui/icons-material/Image";
+import Box from "@mui/material/Box";
+import FormLabel from "@mui/material/FormLabel";
+import Radio, { RadioProps } from "@mui/material/Radio";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import React, { useLayoutEffect, useRef, useState } from "react";
+
+export interface Props {
+  id: string;
+  title: string;
+  description?: string;
+  responseKey?: string | number;
+  img?: string;
+  onChange: RadioProps["onChange"];
+}
+
+const FallbackImage = styled(Box)({
```

Comment:
I spent way too long trying to work out a way to avoid duplicating these styles. As the `Image` is using `component="img"` it's more tricky than it should be.